cpp/test/Ice/slicing/exceptions/Makefile.mk

```diff
@@ -2,7 +2,6 @@
 # Copyright (c) ZeroC, Inc. All rights reserved.
 #
 
-$(test)_sliceflags              := -I$(test)
```

Comment:
We should remove this `-I` in all languages for these tests. This PR does it only for C++.